### ISSUE TITLE
Advent calendar customization

### ISSUE THREAD

**Issue by SedueRey** [otacke/h5p-advent-calendar#16]
I don't know if this is the proper place to add an issue and. then, a PR to try to improve this awesome exercise.

We would want to add this customizations to the project:

- Author would be able to choose the background image and the number of gates.
- The gates lead to text-image or another modules like memory game
- User would have the possibility to return to the home page of the calendar and open another door by pressing a button / keyboard key,

**Comment by otacke** [comment]
@SedueRey Sure, you're at the right place.
But I am not sure if I fully understand what you have in mind.

- You can already choose the background image for both, the whole calendar and/or each door individually if you prefer.
- It's an advent calendar and as such should have 24 doors. Not more, not less. If you want to turn this into a different content type, feel free to bulid one from the code.
- If you want to allow different combinations of content for a popup, you should simply add H5P.Column to the list of libraries in semantics.json and then probably will need to change the resize handling.
- You can already open other doors.

**Comment by SedueRey** [comment]
@otacke our instructors told us they need more or less doors. I know it has a little sense 😅 . If we customize that in a new repository, will that repo work correctly with Wordpress, Drupal or Moodle? I'm sorry if the question is too ... basic, we are trying  to  understand all H5P properly

**Comment by otacke** [comment]
Yes, H5P content types run on any platform that offers an H5P Integration (WordPress, Drupal, Moodle, Typo3, ILIAS, Stud.IP, Lumi, H5P.com, ...) Maybe https://snordian.de/2023/04/22/what-was-that-thing-again-an-h5p-architecture-overview/ is helpful for you.
